config/enums.cwt

```diff
@@ -52,6 +52,267 @@ enums = {
+        wilderness_forest_pine
+        wilderness_mountains
+        wilderness_scope
+        wilderness_steppe
+    }
+    enum[event_environments] = {
+        environment_body
+        environment_council
+        environment_cw_east_main
+        environment_cw_east_spouse
+        environment_cw_east_throneroom_main
+        environment_cw_east_throneroom_spouse
+        environment_cw_india_main
+        environment_cw_india_spouse
+        environment_cw_india_throneroom_main
+        environment_cw_india_throneroom_spouse
+        environment_cw_mediterranean_main
+        environment_cw_mediterranean_spouse
+        environment_cw_mediterranean_throneroom_main
+        environment_cw_mediterranean_throneroom_spouse
+        environment_cw_tavern
+        environment_cw_tavern_spouse
+        environment_cw_tribal_main
+        environment_cw_tribal_spouse
+        environment_cw_west
+        environment_cw_west_spouse
+        environment_event_alley
+        environment_event_alley_day
+        environment_event_armory
+        environment_event_battlefield
+        environment_event_bedchamber
+        environment_event_church
+        environment_event_corridor_day
+        environment_event_courtyard
+        environment_event_desert
+        environment_event_docks
+        environment_event_dungeon
+        environment_event_farms
+        environment_event_feast
+        environment_event_forest
+        environment_event_forest_pine
+        environment_event_gallows
+        environment_event_garden
+        environment_event_genericcamp
+        environment_event_market_east
+        environment_event_market_tribal
+        environment_event_market_west
+        environment_event_mosque
+        environment_event_mountains
+        environment_event_sittingroom
+        environment_event_standard
+        environment_event_steppe
+        environment_event_study
+        environment_event_study_physician
+        environment_event_tavern
+        environment_event_temple
+        environment_event_throne_room_west
+        environment_frontend_east_heir
+        environment_frontend_east_main
+        environment_frontend_east_secondary
+        environment_frontend_india_heir
+        environment_frontend_india_main
+        environment_frontend_india_secondary
+        environment_frontend_mediterranean_heir
+        environment_frontend_mediterranean_main
+        environment_frontend_mediterranean_secondary
+        environment_frontend_tribal_heir
+        environment_frontend_tribal_main
+        environment_frontend_tribal_secondary
+        environment_frontend_west_heir
+        environment_frontend_west_main
+        environment_frontend_west_secondary
+        environment_head
+        environment_hud
+        environment_portrait_editor
+        environment_shoulders
+        environment_standard
+        environment_torso
+        environment_war_overview
+    }
+    enum[animations] = {
```

Comment:
`game\gfx\portraits\portrait_animations`. Again, not sure if the folder matters?